analytics_data_api/v0/models.py

```diff
@@ -445,7 +445,9 @@ class ModuleEngagementMetricRanges(models.Model):
     low_value and high_value.
     """
 
-    course_id = models.CharField(db_index=True, max_length=255)
+    # primary key is actually course_id, metric, and range_type, but only one
+    # primary is allowed in django
+    course_id = models.CharField(primary_key=True, db_index=True, max_length=255)
```

Comment:
I'm curious about the pros and cons to declaring `course_id` the primary key. We tend to use surrogate keys (default in django) everywhere else. I think the result store may be a special case due to its write patterns, but it's still worth considering. In particular, we should be confident that we'll never _ever_ need to change the `course_id` of a record in-between runs of the pipeline, because it has the potential to de-normalize the database. I don't see `primary_key=True` on any other model fields in this file, which leads me to believe that tables are always generated with an `id` field. @mulby do you have any thoughts here? Could we just generate this table with an `id` field?